packages/components/src/font-size-picker/styles.ts

```diff
@@ -19,12 +19,14 @@ export const Container = styled.fieldset`
 `;
 
 export const HeaderLabel = styled( BaseControl.VisualLabel )`
+	display: inline-block;
 	margin-bottom: 0;
 `;
 
 export const HeaderHint = styled.span`
 	color: ${ COLORS.gray[ 700 ] };
-	margin-left: ${ space( 1 ) };
+	display: inline-block;
+	${ rtl( { marginLeft: space( 1 ) }, { marginRight: space( 1 ) } )() }
```

Comment:
_Moar automagic_ in case we kept using this approach: the `rtl` function [should be able](https://github.com/WordPress/gutenberg/blob/44b9b59518a4f04ee1b29e0c1c75b8ffe3e7faa2/packages/components/src/utils/rtl.js#L16-L65) to automatically turn `marginLeft` into `marginRight` !

```suggestion
	${ rtl( { marginLeft: space( 1 ) } )() }
```

Reply:
Neat!